### ISSUE TITLE
Enable interactive login

### ISSUE THREAD

**Issue by sajayantony** [Azure/acr-docker-credential-helper#2]

**Comment by DavidObando** [comment]
Blocked by docker/cli#139.

**Comment by shhsu** [comment]
Also a more minor blocker https://github.com/docker/docker-credential-helpers/pull/69
Note that once this change gets through and propagated into docker cli another PR will be needed to use it to pass environmental variables over.
